kit/score/score.go

```diff
@@ -51,29 +51,29 @@ func (s *Score) DecBy(n int) {
 // Normalize the score to the given maxScore.
 func (s *Score) Normalize(maxScore int) {
 	f := float64(maxScore) / float64(s.MaxScore)
-	normScore := float64(s.Score) * f
+	normScore := float64(s.GetScore()) * f
 	s.Score = int32(math.Round(normScore))
 	s.MaxScore = int32(maxScore)
 }
 
 // weightedScore returns the weighted score for this test score.
 func (s *Score) weightedScore(totalWeight float64) float64 {
-	return weightedScore(float64(s.Score), float64(s.MaxScore), float64(s.Weight), totalWeight)
+	return weightedScore(float64(s.GetScore()), float64(s.GetMaxScore()), float64(s.GetWeight()), totalWeight)
 }
 
 // Equal returns true if s equals other. Ignores the Secret field.
 func (s *Score) Equal(other *Score) bool {
 	return other != nil &&
-		s.TestName == other.TestName &&
-		s.Score == other.Score &&
-		s.MaxScore == other.MaxScore &&
-		s.Weight == other.Weight
+		s.TestName == other.GetTestName() &&
```

Comment:
These can use getters as well as these values are compared, not assigned.

There is an `s.TestName` that can be `s.GetTestName()` on line 88 as well.